spec/unit/pluginHandlers/handlers.spec.js

```diff
@@ -163,6 +163,12 @@ describe('android project handler', function () {
                     'src/android/jniLibs/x86/libnative.so', temp,
                     path.join('app/src/main/jniLibs/x86/libnative.so'), false);
             });
+
+            it('Test#006j : should allow installing sources with target-dir that includes "appco"', function () {
```

Comment:
The bug was that it included `app`, right?
```suggestion
            it('Test#006j : should allow installing sources with target-dir that includes "app"', function () {
```